src/cljs/rems/administration/resources.cljs

```diff
@@ -45,6 +59,63 @@
    {:href (str "/#/administration/resources/" resource-id)}
    (text :t.administration/view)])
 
+
+;;; Archiving
+;; TODO: deduplicate
```

Comment:
Do you have ideas that how to remove this duplication between resource and catalogue pages?

Reply:
re-frame is a framework that does not support this kind of reusable components well. However refactoring this into Reagent components does not really help.

One option is to extract a common enable/disable as well as an archived ns and then parametrize each handler,  subscription and button there with the local key as well. I.e. `::set-display-archived?` subscription in `administration/archived_item.clj` would accept as a parameter the item prefix (i.e. `::resource`, `::catalogue-item`) and store and retrieve in the db under the path `[::resource ::archived]` (or the reverse if it makes more sense). Same for handlers and buttons.